vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -597,8 +603,10 @@ void ViewViewManager::SyncFocusableAndAccessible(ViewShadowNode *pViewShadowNode
     const auto isFocusable = pViewShadowNode->IsFocusable();
     const auto isAccessible = pViewShadowNode->IsAccessible();
     const auto isPressable = pViewShadowNode->OnClick();
+    const auto isDisabled = pViewShadowNode->IsDisable();
     const auto isTabStop =
-        (isPressable && isFocusable && isAccessible) || (!isPressable && (isFocusable || isAccessible));
+        (!isDisabled &&
```

Comment:
If you haven't already, I'd also double check that this issue (https://github.com/microsoft/react-native-windows/issues/10687) isn't being reintroduced just with the disabled prop causing it now